colossalai/shardformer/shard/shardformer.py

```diff
@@ -32,7 +30,7 @@ class ShardFormer:
     """
 
     def __init__(self, shard_config: ShardConfig):
-        self.coordinator = DistCoordinator()
+        # self.coordinator = DistCoordinator()
```

Comment:
Not a good idea to simply remove it here.

Reply:
Will discuss how to revise this func later

Reply:
revised